pkg/operator/ceph/cluster/cluster.go

```diff
@@ -116,7 +119,13 @@ func (c *cluster) createInstance(rookImage string, cephVersion cephver.CephVersi
 func (c *cluster) doOrchestration(rookImage string, cephVersion cephver.CephVersion, spec *cephv1.ClusterSpec) error {
 	// Create a configmap for overriding ceph config settings
 	// These settings should only be modified by a user after they are initialized
-	err := populateConfigOverrideConfigMap(c.context, c.Namespace, c.ownerRef)
+	var configOverrideValue string
+	if spec.IPFamily == cephv1.IPv6 {
+		configOverrideValue = strings.TrimSpace(bindIpv6Address)
+		// TODO: Check if cluster is actually bound to Ipv6 address. (How?)
```

Comment:
per discussion, let's not worry about this check, they will just see an error when the mons try to start